implementations/code_df/issues_active_github.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime
+import pandas as pd
+
+from implementations.code_df.utils import read_json_file, str_to_date
+from implementations.code_df.issue_github import IssueGitHub
+
+
+class IssuesActiveGitHub(IssueGitHub):
+    """
+    Class for Issues Active
+    """
+
+    def _flatten(self, item):
+        """
+        Flatten a raw issue fetched by Perceval into a flat dictionary.
+
+        A list with a single flat directory will be returned.
+        That dictionary will have the elements we need for computing metrics.
+        The list may be empty, if for some reason the commit should not
+        be considered.
+
+        :param item: raw item fetched by Perceval (dictionary)
+        :returns:   list of a single flat dictionary
+        """
+
+        creation_date = str_to_date(item['data']['created_at'])
+        updation_date = str_to_date(item['data']['updated_at'])
+
+        # considering only those issues who were updated
+        # within the given period
+
+        if self.since and (self.since > creation_date) \
+                and (self.since > updation_date):
+            return []
+
+        if self.until and (self.until < creation_date) \
+                and (self.until < updation_date):
+            return []
+
+        flat = {
+            'repo': item['origin'],
+            'hash': item['data']['id'],
+            'category': "issue",
+            'author': item['data']['user']['login'],
+            'created_date': creation_date,
+            'current_status': item['data']['state'],
+            'updated_date': updation_date
+        }
+
+        return [flat]
+
+    def compute(self):
```

Comment:
I don't think this works. We're defining "active issues" as those that had some change in state or comment during the period, and this is not capturing that... Have in mind that "updated_date", if I'm not wrong, is giving only the last update to the issue...

I think you need to explore Perceval, to see how this information can be captured, I think it is not captured by default.

If needed, maybe @valeriocos can help here.

Reply:
@jgbarah, I was going to discuss this in today's meeting. As far as I know, Perceval does not handle events too well. You're right about the `updated_date` meaning. I have opened an issue in Perceval which mentions this, but not specifically. I'll reword that issue or open a new one.

Reply:
OK, let's discuss today.

Reply:
I think issues in Perceval are still not solved. @Polaris000, if I'm right, what do you think about closing this pr? When Perceval is ready, we could produce a new patch: it will very likely be different from this one, anyway...

Reply:
Sure. Closing it for now..